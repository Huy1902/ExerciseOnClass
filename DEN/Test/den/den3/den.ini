[Compile]
CompileMode=DEBUG

[Help]
Files=""

[Editor]
DefaultTabSize=8
DefaultIndentSize=1
DefaultFlags=20599
DefaultSaveExt=.pas

[Highlight]
Exts="*.pas;*.pp;*.inc"
NeedsTabs="make*;make*.*"

[SourcePath]
SourceList=";C:\FPC\2.0.4\bin\i386-win32\"

[Mouse]
DoubleDelay=8
ReverseButtons=0
AltClickAction=6
CtrlClickAction=1

[Keyboard]
EditKeys=borland

[Search]
FindFlags=4

[Breakpoints]
Count=1
Type0=file-line
State0=disabled
FileName0=c:\fpc\2.0.4\bin\i386-win32\bai5_1.pas
LineNumber0=59

[Watches]
Count=0

[Preferences]
DesktopFileFlags=209
CenterCurrentLineWhileDebugging=1
AutoSaveFlags=7
MiscOptions=6
DesktopLocation=1

[Misc]
ShowReadme=0

[Files]
OpenExts="*.pas;*.pp;*.inc"
RecentFile1=C:\FPC\2.0.4\bin\i386-win32\daoday.out,0,0
RecentFile2=C:\FPC\2.0.4\bin\i386-win32\daoday.inp,0,0
RecentFile3=C:\FPC\2.0.4\bin\i386-win32\hh.inp,3,0
RecentFile4=C:\FPC\2.0.4\bin\i386-win32\gonbac.pas,0,0
RecentFile5=C:\FPC\2.0.4\bin\i386-win32\xe.out,0,0
RecentFile6=C:\FPC\2.0.4\bin\i386-win32\xe.inp,13,3
RecentFile7=C:\FPC\2.0.4\bin\i386-win32\hh.out,0,0
RecentFile8=C:\FPC\2.0.4\bin\i386-win32\mh.out,0,0
RecentFile9=C:\FPC\2.0.4\bin\i386-win32\mh.inp,0,0
PrinterDevice=prn
