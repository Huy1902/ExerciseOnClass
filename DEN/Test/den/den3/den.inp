1 10000
5000 10000