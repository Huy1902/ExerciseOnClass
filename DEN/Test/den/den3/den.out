189360
100526
